guides/guides.js

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env node
+
+/*
+ * Copyright 2024 Red Hat Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+const https = require('follow-redirects').https
```

Comment:
Instead of using a third party dependency to make requests you can use the [Fetch API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API), which has a promise-based API and allows [following redirects](https://developer.mozilla.org/en-US/docs/Web/API/Request/redirect) if needed.